content/docs/reference/config/authorization/constraints-and-properties/index.md

```diff
@@ -31,8 +31,14 @@ The following table lists the currently supported keys for the `constraints` fie
 | `destination.name` | Destination workload instance name | YES | `destination.name` | `["productpage*", "*-test"]` |
 | `destination.namespace` | Destination workload instance namespace | YES | `destination.namespace` | `["default"]` |
 | `destination.user` | The identity of the destination workload | YES | `destination.user` | `["bookinfo-productpage"]` |
+| `experimental` | Experimental configuration, Values wrapped in `[]` are matched as a list | YES | `experimental.envoy.filters.network.mysql_proxy[db.table]` | `["[update]"]` |
```

Comment:
Let's document "experimental.envoy.filters.*". This is how we do the matching in code.

Reply:
Addressed in e84f448.